training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.3-gpu/context/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.0' \
```

Comment:
Please consider moving these and the packages below to a requirements.txt file and install with one call to `pip install`.

Reply:
what is the benefit of moving to requirements.txt?

Reply:
It's more readable.

Reply:
In docker, do we just keep the base image?

Reply:
Sorry, I don't know how the base image would relate to a requirements.txt file. I'm just suggesting that you could have something like this in the Dockerfile:

```
COPY requirements.txt .
RUN pip install -r requirements.txt --no-cache-dir && rm requirements.txt
```

and requirements.txt would contain something like:
```
ipykernel~=6.0
azureml-core=={{latest-pypi-version}}
<and other packages>
```